charts/edc-dataplane/README.md

```diff
@@ -9,7 +9,7 @@ EDC Data-Plane - The Eclipse DataSpaceConnector data layer with responsibility o
 ## TL;DR
 ```shell
 $ helm repo add catenax-ng-product-edc https://catenax-ng.github.io/product-edc
-$ helm install my-release catenax-ng-product-edc/edc-dataplane --version 0.1.2
+$ helm install my-release catenax-ng-product-edc/edc-dataplane --version 0.1.5
```

Comment:
Do not use references to catenax-ng GH organization in Eclipse-Tractusx. For your example, you can use our central Helm repository available at https://eclipse-tractusx.github.io/charts/dev:

```
$ helm repo list
NAME        	URL
tractusx-dev	https://eclipse-tractusx.github.io/charts/dev
$ helm search repo edc
NAME                         	CHART VERSION	APP VERSION	DESCRIPTION
tractusx-dev/edc-controlplane	0.1.2        	0.1.2      	EDC Control-Plane - The Eclipse DataSpaceConnec...
tractusx-dev/edc-dataplane   	0.1.2        	0.1.2      	EDC Data-Plane - The Eclipse DataSpaceConnector...
```